dev/com.ibm.ws.cdi.2.0_fat/test-applications/secureAsyncEventsApp/src/secureAsyncEventsApp/web/SecureAsyncEventsServlet.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2017, 2018 IBM Corporation and others.
```

Comment:
delete 2017

Reply:
I think I need both dates, the classes were originally delivered under other PRs in 2017. I have now amended them in 2018.